### PR TITLE
Release note for 4.14.2

### PR DESCRIPTION
**Pull request by zymap**
---

*Motivation*

Release note update for 4.14.2

### CODE REVIEW

**Comment by zymap** [comment]
@eolivelli Here has an update for the site. PTAL. Thanks.

File: site/docs/4.14.2/overview/releaseNotes.md [hunk 1]
```diff
@@ -20,6 +20,22 @@ The technical details of this release are summarized below.
 
   The current libthrift version 0.12.0 has multiple vulnerabilities: CVE-2019-0205 , CVE-2019-0210 , CVE-2020-13949
 
+- [https://github.com/apache/bookkeeper/pull/2735] Exclude grpc-okhttp dependency
+
+  The okhttp dependency version 2.7.4 is old and vulnerable. This dependency isn't needed and it causes Bookkeeper to be flagged for security vulnerabilities.
+
+- [https://github.com/apache/bookkeeper/pull/2734] Upgrade Freebuilder version and fix the dependency
+
+  - Freebuilder 1.14.9 contains an outdate jquery js file which causes the library to be flagged as vulnerable with the highest threat level in Sonatype IQ vulnerability scanner. This also flags Bookkeeper and Pulsar as vulnerable with the highest threat level although it is a false positive and not an actual threat.
```
Comment (nicoloboschi) [inline site/docs/4.14.2/overview/releaseNotes.md:29]: idk if it's good to mention pulsar here, I feel it is not relevant for BK release notes

**Review by nicoloboschi — CHANGES_REQUESTED** [review]
I think we are missing this important commit about bc-fips dependency fix
https://github.com/apache/bookkeeper/commit/e54be3451726c6b7e2fe6a9d412f1a2ab78d8203

File: site/docs/4.14.2/overview/releaseNotes.md (hunk 1 above)
Comment (eolivelli) [reply site/docs/4.14.2/overview/releaseNotes.md:29]: yes, please remove the reference to Pulsar @zymap

**Review by eolivelli — CHANGES_REQUESTED** [review]

**Comment by zymap** [comment]
@eolivelli Done. PTAL thanks

**Review by eolivelli — APPROVED** [review]
Lgtm

**Comment by nicoloboschi** [comment]
> I think we are missing this important commit about bc-fips dependency fix
> [e54be34](https://github.com/apache/bookkeeper/commit/e54be3451726c6b7e2fe6a9d412f1a2ab78d8203)

@zymap @eolivelli I see the RN merged but my comment isn't resolved yet

**Comment by eolivelli** [comment]
@nicoloboschi you are right.
I apologise.

@zymap can you please address @nicoloboschi 's comments in a new patch?

**Comment by zymap** [comment]
We have added it in the rc0 release note https://github.com/apache/bookkeeper/pull/2763/files#diff-e0a1e7642b2b55b58ef495600503f7c31d4d4e4c4c48d4c176110edb0ead0274R11

**Pull request merged**